src/styles/main.scss

```diff
@@ -1,3 +1,69 @@
-body {
-  margin: 0;
+$cell-size: 100px;
+
+html {
+  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $cell-size * 7 + 6px;
+  padding: 10px;
+  gap: 1px;
+
+  & :first-child {
+    margin-left: $cell-size * 6 + 6px;
```

Comment:
margin-left should depends on `start-day` modifier
for example on screen set wed, but layout not change
![Screenshot from 2023-05-21 18-09-41](https://github.com/mate-academy/layout_calendar/assets/33029864/d3094f95-0f77-4854-b6ed-cec4f047f8b2)